src/routes/dashboard/csv/downloadCSV.ts

```diff
@@ -0,0 +1,115 @@
+import type { Questions } from '../types'
+
+export function toCSV(channels: string[], questions: Map<string, Questions>) {
+  let csv =
+    'Channel,Number of Questions, Answered by Staff, Answered by Community, Unanswered,'
+
+  const sortedQs: Map<string, Map<string, Questions>> = reshape(
+    channels,
+    questions
+  )
+
+  const [first] = sortedQs.values()
+  if(!first) return /** @todo throw some error */
+  const columns = first.keys()
+  csv += Array.from(columns).join(',')
+  csv += '\n'
+
+  sortedQs.forEach((datesMap, channel) => {
```

Comment:
minor code style suggestion, elsewhere in the codebase we're using `for...of`, could this be adjusted to something like

```ts
for (const [datesMap, channe] of sortedQs.entries()) {}
```